src/RandArray.chpl

```diff
@@ -14,30 +15,21 @@ module RandArray {
   private config const logLevel = ServerConfig.logLevel;
   const raLogger = new Logger(logLevel);
 
-  proc fillInt(a:[] ?t, const aMin, const aMax, const seedStr:string="None") throws where isIntType(t) {
+  proc fillInt(a:[] ?t, const aMin: t, const aMax: t, const seedStr:string="None") throws where isIntType(t) {
       if (seedStr.toLower() == "none") {
-        fillRandom(a);
+        fillRandom(a, aMin, aMax-1);
```

Comment:
this is a good idea! I think this will make it much easier to track these back down if chapel 1.28 does make the change to exclusive upper bounds like some of the comments in this PR suggest

Reply:
Added

Reply:
w.r.t. the potential change in Chapel 1.28, I'm not sure I ever mentioned that I opened an issue about changing the upper bound to exclusive in https://github.com/chapel-lang/chapel/issues/20165.  As noted there, a challenge will be how to get users from the old behavior to the new without surprises or too much annoyance.